servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -88,8 +88,19 @@ def get_job(id, **kwargs):
 
     :rtype: JobMetadataResponse
     """
-    url = '{cromwell_url}/{id}/metadata'.format(
-        cromwell_url=_get_base_url(), id=id)
+
+    include_keys = [
```

Comment:
optional: how about making this a `set([])`or a tuple than a list?

Reply:
ok – made it into a tuple